### PR TITLE
Feature/functionality

### LINKED ISSUE #20: Add Gender to User Registration
As a user, I should be asked for a gender when creating an account. This information should be integrated into the existing user data and database.

### PR DESCRIPTION
**Pull request by michellekaplan7**
## All Submissions:
Completed FE functionality for our track and added some tests

### New Feature Submissions:

* [X] Have you added an explanation of what your changes do?
* [X] Have you lint your code locally prior to submission?

### CODE REVIEW

File: code/api/src/seeders/3-crates.js [hunk 1]
```diff
@@ -4,41 +4,23 @@ module.exports = {
...
-        description: 'A monthly supply of trendy clothes for women.',
+        name: 'Accessories',
+        description: 'A monthly supply of trendy accessories.',
         createdAt: new Date(),
         updatedAt: new Date()
       },
       {
-        name: 'Accessories for Men',
-        description: 'A monthly supply of trendy accessories for men',
+        name: 'Clothes and Accessories',
```
Comment (kristastadler) [inline code/api/src/seeders/3-crates.js:19]: I like that you removed the extra seeders since including gender in the name seems irrelevant right now.

**Review by tylertomlinson — APPROVED** [review]
Nice. The PR very descriptive. Didn't even have to look the files 🥇

File: code/web/src/modules/user/Signup/Signup.test.js [hunk 2]
```diff
@@ -1,46 +1,23 @@
...
-import * as actions from '../api/actions'
+import Signup from "./Signup";
+import { MemoryRouter, BrowserRouter } from "react-router-dom";
+import { createStore } from "redux";
+import { Provider } from "react-redux";
+import { rootReducer } from "../../../setup/store";
 
 describe("Signup", () => {
-  it.skip('should record a users name, email, password and gender', () => {
+  it("should record a users name, email, password and gender", () => {
```
Comment (kristastadler) [inline code/web/src/modules/user/Signup/Signup.test.js:11]: Nice job adding testing!

**Review by kristastadler — APPROVED** [review]
